src/DIRAC/FrameworkSystem/Client/ProxyManagerClient.py

```diff
@@ -206,12 +206,12 @@ def uploadProxy(self, proxy=None, restrictLifeTime=0, rfcIfPossible=False):
     # Make sure it's valid
     if chain.hasExpired().get('Value'):
       return S_ERROR("Proxy %s has expired" % proxyLocation)
-    if chain.getDIRACGroup().get('Value') or chain.isVOMS().get('Value'):
+    if chain.getDIRACGroup(ignoreDefault=True).get('Value') or chain.isVOMS().get('Value'):
       return S_ERROR("Cannot upload proxy with DIRAC group or VOMS extensions")
 
     rpcClient = RPCClient("Framework/ProxyManager", timeout=120)
     # Get a delegation request
-    result = rpcClient.requestDelegationUpload(chain.getRemainingSecs()['Value'])
+    result = rpcClient.requestDelegationUpload(chain.getRemainingSecs()['Value'], None)
```

Comment:
I see this warning:

https://github.com/DIRACGrid/DIRAC/blob/e142b36e885ae2b5490ad6c13fcd543a770696de/src/DIRAC/FrameworkSystem/Service/ProxyManagerHandler.py#L95-L99

So I would not touch this line.

Reply:
Well, this will still work, but I will remove it. I needed it in our weird setting of having to run v7r1 service because of some our users still at the old client version.